src/test/java/ClassTest.java

```diff
@@ -1,80 +1,50 @@
+        Assert.assertNotEquals(-1, numberOfCharacters);
+        Assert.assertNotEquals(0,numberOfCharacters);
+
     }
 
     @Test
-    public void WrongFlagAndGoodFile() {
+    public void fileDoesntExist() {
 
-        try {
-            WordCounter wordCounter = new WordCounter("test.txt", "incorrect flag");
-            int wrongFlag = wordCounter.Count();
-            Assert.assertEquals(wrongFlag, -1);
+            Counter counter = new Counter();
+            int numberOfCharacters = counter.countCharacters(new File("/Users/wrong_paths.txt"));
+            Assert.assertEquals(numberOfCharacters,-1);
 
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
     }
 
     @Test
-    public void WrongFileAndGoodFlag() {
-
-        try {
-            WordCounter wordCounter = new WordCounter("x.txt", "w");
-            int wrongFile = wordCounter.Count();
-            Assert.assertEquals(wrongFile, -1);
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
-    }
+    public void checkTestFile() {
 
-    @Test
-    public void CheckTestFile() {
-
-        try {
-            WordCounter wordCounterLine = new WordCounter("test.txt", "l");
-            int numberOfLines = wordCounterLine.Count();
-            WordCounter wordCounterWords = new WordCounter("test.txt", "w");
-            int numberOfWords = wordCounterWords.Count();
-            WordCounter wordCounterCharacters = new WordCounter("test.txt", "c");
-            int numberOfCharacters = wordCounterCharacters.Count();
+            Counter counter = new Counter();
+            int numberOfLines = counter.countLines(new File("/Users/mzyrkowski/IdeaProjects/Ważne - zadania/007_Word_Counter_New/src/test/java/testFileWith3Lines5Words9Characters.txt"));
+            int numberOfWords = counter.countWords(new File("/Users/mzyrkowski/IdeaProjects/Ważne - zadania/007_Word_Counter_New/src/test/java/testFileWith3Lines5Words9Characters.txt"));
+            int numberOfCharacters = counter.countCharacters(new File("/Users/mzyrkowski/IdeaProjects/Ważne - zadania/007_Word_Counter_New/src/test/java/testFileWith3Lines5Words9Characters.txt"));
 
             Assert.assertEquals(numberOfLines, 3);
             Assert.assertEquals(numberOfWords, 5);
             Assert.assertEquals(numberOfCharacters, 9);
 
-        } catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
+    }
+
+    @Test
+    public void existingEmptyFile(){
+        Counter counter = new Counter();
+        int numberOfCharacters = counter.countCharacters(new File("/Users/mzyrkowski/IdeaProjects/Ważne - zadania/007_Word_Counter_New/src/test/java/emptyFile.txt"));
+        Assert.assertEquals(0,numberOfCharacters);
+        Assert.assertNotEquals(-1,numberOfCharacters);
```

Comment:
jesli masz juz assert Equals ktory przejdzie to nie ma sensu robic notEquals. To jest duplikacja tego samego i nie jest potrzebna.